### PR TITLE
Add policy to grant access to static.synapse.org

### PR DESCRIPTION
**Pull request by nickgros**
And also give that access to me because I can't upload to this bucket and I need to

### CODE REVIEW

File: templates/accounts.yaml [hunk 1]
```diff
@@ -136,6 +137,11 @@ Resources:
         - arn:aws:iam::aws:policy/AmazonAthenaFullAccess
         - arn:aws:iam::aws:policy/AWSGlueConsoleFullAccess
         - !Ref AWSIAMAnalyticsUsersAccessPolicy
+  AWSIAMStaticSynapseUsersGroup:
```
Comment (ahayden) [inline templates/accounts.yaml:140]: Is it reasonable to rename this group something like `AWSIAMWebEngineerUsersGroup`?

**Review by ahayden — CHANGES_REQUESTED** [review]
Looks fine, just one question.... Also, please comment is there's a ticket number for this.

File: templates/accounts.yaml (hunk 1 above)
Comment (nickgros) [reply templates/accounts.yaml:140]: Good idea. Will make that change

**Comment by nickgros** [comment]
No ticket number. Should I create a ticket?

**Comment by zaro0508** [comment]
instead of adding more AWS users/groups why don't we just work on the existing PR federate jumpcloud users/group into this account? https://github.com/Sage-Bionetworks/synapseprod-infra/pull/55

**Comment by ahayden** [comment]
Let's get a perspective from Bruce on going the federation route instead

**Comment by nickgros** [comment]
How close is the JC federation to being applied? This is a blocker for me. Could we add the proposed group and then remove it later?

**Review by ahayden — APPROVED** [review]
Approving as necessary to unblock work before migrating to federated access

**Comment by nickgros** [comment]
@ahayden Not sure if you are waiting on @brucehoff 's review, but I don't have permission to merge this if you think it is ok to merge.

**Comment by brucehoff** [comment]
I'm curious why @nickgros (or any individual engineer) needs access to a production bucket.  If the bucket contains static content that is part of Synapse then shouldn't upload to the bucket be done by an automated CI/CD process rather than by an engineer?  I.e., shouldn't there be a Jenkins job running on an EC2 that has a _role_ that lets it deploy content to the bucket?

**Pull request merged**